contrib/azurerm/README.md

```diff
@@ -0,0 +1,64 @@
+# Kubernetes on Azure with Azure Resource Group Templates
```

Comment:
The readme should be linked or moved into the main `/doc` to make it easy for people to find it.